agave_app/main.cpp

```diff
@@ -159,9 +160,14 @@ main(int argc, char* argv[])
 
     result = a.exec();
   } else {
-    agaveGui* w = new agaveGui();
-    w->show();
-    result = a.exec();
+    try {
+      agaveGui* w = new agaveGui();
+      w->show();
+      result = a.exec();
+

```

Comment:
(nit) intentional whitespace?    Surprised a linter didn't catch this.

Reply:
The curly brackets of the try block caused one more level of indentation. Is that what you're seeing?